modules/nobidBidAdapter.js

```diff
@@ -118,6 +118,7 @@ function nobidBuildRequests(bids, bidderRequest) {
     state['ref'] = document.referrer;
     state['gdpr'] = gdprConsent(bidderRequest);
     state['usp'] = uspConsent(bidderRequest);
+    state['rfc'] = window.nobid.refreshCount;
```

Comment:
I think this should check to make sure the nobid object is defined before accessing refreshCount.

Please add a unit test as well.